deps/rabbitmq_peer_discovery_consul/priv/schema/rabbitmq_peer_discovery_consul.schema

```diff
@@ -231,20 +231,6 @@ fun(Conf) ->
 end}.
 
 
-%% use long node names?
-
-{mapping, "cluster_formation.consul.use_longname", "rabbit.cluster_formation.peer_discovery_consul.consul_use_longname", [
```

Comment:
If we delete these settings, it can only go into 3.12.0. Even then I'd leave them as no-ops for a period of time.

Reply:
Restored old versions of schema and hrl files.